autosubmit/job/job_list.py

```diff
@@ -1377,14 +1370,17 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
             if dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "chunk":
                 if aux_key in distances_of_current_section and distance > distances_of_current_section[aux_key]:
                     distances_of_current_section[aux_key] = distance
-            elif dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "member":
-                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[aux_key]:
-                    distances_of_current_section_member[aux_key] = distance
+
+            elif dic_jobs.as_conf.jobs_data.get(aux_key, { }).get("RUNNING", "once") == "member":
+                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[
+                    aux_key]:
+                    distances_of_current_section_member[aux_key]=distance
```

Comment:
This introduced a couple of warnings in PyCharm. First, the `aux_key]` in the next line shows as wrong indentation (the `[` normally implies the next item will come in the same column).

Maybe it'd be easier to simplify the `if` statement into a single one. Maybe something like `if distances_of_current_section_member.get(aux_key, False):`

And below, it's missing a space around `=`.